compiler/shared/test/diff-ir/IRRec.mls

```diff
@@ -1,6 +1,7 @@
 :NewParser
 :ParseOnly
 :UseIR
+:NoTailRec
```

Comment:
When this is enabled, don't duplicate the IR printout.

Reply:
resolved in latest commit

Reply:
resolved in latest commit